src/patternfly/components/ContextSelector/examples/context-selector-example.hbs

```diff
@@ -11,7 +11,7 @@
       {{#> input-group}}
         {{#> form-control controlType="input" input="true" form-control--attribute=(concat 'type="search"' 'placeholder="Search"' 'id="textInput1"' 'name="textInput1"' 'aria-labelledby="' context-selector--id '-search-button"')}}
         {{/form-control}}
-        {{#> button button--modifier="pf-m-tertiary" button--attribute=(concat 'id="' context-selector--id '-search-button"' 'aria-label="Search menu items"')}}
+        {{#> button button--modifier="pf-m-input" button--attribute=(concat 'id="' context-selector--id '-search-button"' 'aria-label="Search menu items"')}}
```

Comment:
the context selector and input group examples should use `pf-m-control` instead of `pf-m-input`